pkg/webui/containers/side-bar/side-bar.styl

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.sidebar
+  border-normal(right)
+  line-height: 1
+  font-size: $fs.s
+  height: 97.5vh
+  min-width: 19.8rem
+  +media-query($bp.m)
+    sidebar-transition(left, box-shadow)
+    top: $header-height-mobile
+    left: -60vw
+    bottom: 0
+    width: 60vw
+    max-width: 20rem
+
+    &-open
+      box-shadow: 0 0 13rem 2rem rgba(0,0,0,.4)
+      left: 0
+
+  &-minimized
+    width: 4rem
+    min-width: 4rem
+    max-width: 4rem
```

Comment:
```suggestion
    width: 4rem
```

Reply:
Was there an issue with this? If you set `width` with an absolute value, it should make min/max-width superfluous.